IdentityCore/src/webview/embeddedWebview/ui/MSIDWebviewUIController.h

```diff
@@ -25,6 +25,7 @@
 
 #import <Foundation/Foundation.h>
 #import <WebKit/WebKit.h>
+#import "MSIDWorkPlaceJoinConstants.h"
```

Comment:
is this needed here?

Reply:
Technically, we can put in in the MSIDWebviewUIController.m file, but this is a project target header, so I don't see any major concerns. It is needed in the implementation file to recognize the PKeyAuth constant variable.

Reply:
Yes, let's move it to .m file :)